model/coverage.py

```diff
@@ -242,6 +242,13 @@ def update(self, start=None, finish=None, achievements=None,
         # .exception
         self.exception = exception
 
+    def to_data(self):
+        """Convert this Timestamp to an unfinalized TimestampData."""
+        from ..metadata_layer import TimestampData
```

Comment:
What does the double dots mean for this import statement?

Reply:
We're in the `core.model` package, and we need to import something from the `core` package above. It's analogous to `..` in a shell referring to the parent directory.

This is also why the import statement is inside the function (executed at runtime), rather than at the top of the file (executed on startup). `metadata_layer.py` requires model code to run, and trying to import `metadata_layer` while the model code is loading would create a loop.